partiql-eval/src/main/kotlin/org/partiql/lang/eval/internal/SequenceExprValue.kt

```diff
@@ -0,0 +1,44 @@
+package org.partiql.lang.eval.internal
+
+import org.partiql.lang.eval.BaseExprValue
+import org.partiql.lang.eval.ExprValue
+import org.partiql.lang.eval.ExprValueType
+import org.partiql.lang.eval.OrdinalBindings
+import org.partiql.lang.eval.internal.ext.namedValue
+
+internal class ListExprValue(val values: Sequence<ExprValue>) : BaseExprValue() {
```

Comment:
These sequence `ExprValue`s were ported from `ExprValue.kt`. They were internal to `partiql-lang`. Needed to case on the concrete Java class name in `ExcludeRelationalOperatorFactory`, so ported to this file in `partiql-eval`.